site/content/blogs/release-v126.en.md

```diff
@@ -37,6 +37,47 @@ Copilot v1.26 brings several new features and improvements:
 ## `storage init` for environment addons
 
 ## Sidecar image build
+You can now build Sidecar container images natively from a Dockerfile. Copilot builds and pushes multiple sidecar container images in parallel
+to the same ECR repository of your main container.
+You can modify your workload manifest as below.
```

Comment:
```suggestion
You can now use Copilot to build sidecar container images natively from a Dockerfile, and get them pushed to ECR.
```

Let's not mention parallel building/pushing until the PR gets merged.
You can modify your workload manifest as below.